modules/customize-certificates-replace-default-router.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="replacing-default-ingress-{context}"]
+= Replacing the default ingress certificate
+
+You can replace the default ingress certificate for all
+applications under the `.apps` subdomain. After you replace
+the certificate, all applications, including the web console
+and CLI, will have encryption provided by specified certificate.
+
+.Prerequisites
+
+* You must have a wildcard certificate and its private key, 
+both in the PEM format, for use. 
+* The certificate must have a `subjectAltName` extension of 
+`*apps.<clustername>.<domain>`.
+
+.Procedure
+
+. Create a secret that contains the wildcard certificate and key:
++
+----
+$ oc create secret tls <certificate> \//<1>
+     --cert=</path/to/cert.crt> \//<2>
+     --key=</path/to/cert.key> \//<3>
+     -n openshift-ingress
+----
+<1> `<certificate>` is the name of the secret that will contain
+the certificate and private key.
+<2> `</path/to/cert.crt>` is the path to the certificate on your
+local file system.
+<3> `</path/to/cert.key>` is the path to the private key associated
+with this certificate.
+
+. Update the ingress controller configuration with the newly created
+secret:
++
+----
+$ oc patch ingresscontroller.operator cluster \
```

Comment:
should be `oc patch ingresscontroller.operator default`, because the default ingresscontroller has been renamed from `cluster` to `default`.

Reply:
This has been corrected.